docs/ERROR_CODES.md

```
@@ -15,6 +15,11 @@ All compiler errors use stable, documented error codes. Never ignore or suppress
 **Message**: Input source is empty
 **Fix**: Provide non-empty SDL or canonical AST JSON input
 
+### `SVJIF_E_INPUT_INVALID_JSON`
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)